### PR TITLE
Remove API password

### LINKED ISSUE #289: Remove password field from project resource in API
I see little purpose (even less after #286 ) exposing the project password hash in the API, like it appears here:

`GET http://localhost:5000/api/projects/demo`
```json
{
    "contact_email": "demo@example.com",
    "members": [
        {
            "activated": true,
            "weight": 1.0,
            "name": "aaa",
            "id": 1
        },
        {
            "activated": true,
            "weight": 1.0,
            "name": "bbb",
            "id": 2
        }
    ],
    "balance": {
        "1": 6.0,
        "2": -6.0
    },
    "name": "demo",
    "password": "pbkdf2:sha256:50000$NQkl8AHg$6efe065565e51ead5755f8c6847aff34df547fd11652227f68cae46587fc8e5e",
    "id": "demo",
    "active_members": [
        {
            "activated": true,
            "weight": 1.0,
            "name": "aaa",
            "id": 1
        },
        {
            "activated": true,
            "weight": 1.0,
            "name": "bbb",
            "id": 2
        }
    ]
}
```

What about removing this field from DB output ?

### PR DESCRIPTION
**Pull request by JocelynDelalande**
fix #289

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -12,6 +12,7 @@ Breaking changes
 - ``ADMIN_PASSWORD`` is now stored hashed. The ``ihatemoney generate_password_hash`` command can now be used to generate a proper password HASH (#236)
 - Turn the WSGI file into a python module, renamed from budget/ihatemoney.wsgi to ihatemoney/wsgi.py. Please update your Apache/Gunicorn configuration! (#218)
 - Admin privileges are now required to access the dashboard (#262)
+- `password` field has been removed from project API GET views (#289)
```
Comment (almet) [inline CHANGELOG.rst]: If this is considered a breaking change, maybe should we indicate it here?

**Comment by almet** [comment]
That looks good to me. Feel free to merge yourself once my minor comment has been addressed.

File: CHANGELOG.rst (hunk 1 above)
Comment (JocelynDelalande) [reply CHANGELOG.rst]: That is why I put it in the "breaking changes" section :-). Were you considering some other way to indicating it ?

Comment (almet) [reply CHANGELOG.rst]: Ah, sorry, it's all good then :)

**Pull request merged**